importer/importer_test.go

```diff
@@ -420,7 +418,7 @@ func TestImporter_Validate(t *testing.T) {
 	t.Parallel()
 
 	// This test is particularly useful when DEBUG is set to show diff validate() errors. So force it on/off.
-	Logger.SetLevel(log.DEBUG)
+	Logger.SetLevel(4)
```

Comment:
which Logger does this use?

Reply:
https://github.com/CycloneDX/sbom-utility/blob/86071171c03147419814decec536cd68bf05d48b/log/log.go#L47-L53

Reply:
https://github.com/CycloneDX/license-scanner/blob/main/importer/importer.go#L21